sysapps/device_capabilities/cpu_info_provider_android.cc

```diff
@@ -20,6 +20,8 @@ const char kProcLoadavg[] = "/proc/loadavg";
 namespace xwalk {
 namespace sysapps {
 
+void CPUInfoProvider::init() {}
```

Comment:
Well there is no Win-specific constructor, the _win file is not a subclass if CPUInfoProvider. So what I can do is ifdef out the constructer in cpu_info_provider.cc only for Windows and implement the CPUInfoProvider constructor in the _win class. The other solution is to put the few init() lines inside cpu_info_provider.cc directly protected by a ifdef only on Windows, but then again what's the point of a _win file in the first place.